modules/caddyhttp/fileserver/browsetpl.go

```diff
@@ -14,467 +14,7 @@
-							{{- end}}
-						</th>
-						<th class="hideable"></th>
-					</tr>
-					</thead>
-					<tbody>
-					{{- if .CanGoUp}}
-					<tr>
-						<td></td>
-						<td>
-							<a href="..">
-								<span class="goup">Go up</span>
-							</a>
-						</td>
-						<td>&mdash;</td>
-						<td class="hideable">&mdash;</td>
-						<td class="hideable"></td>
-					</tr>
-					{{- end}}
-					{{- range .Items}}
-					<tr class="file">
-						<td></td>
-						<td>
-							<a href="{{html .URL}}">
-								{{- if .IsDir}}
-								<svg width="1.5em" height="1em" version="1.1" viewBox="0 0 317 259"><use xlink:href="#folder{{if .IsSymlink}}-shortcut{{end}}"></use></svg>
-								{{- else}}
-								<svg width="1.5em" height="1em" version="1.1" viewBox="0 0 265 323"><use xlink:href="#file{{if .IsSymlink}}-shortcut{{end}}"></use></svg>
-								{{- end}}
-								<span class="name">{{html .Name}}</span>
-							</a>
-						</td>
-						{{- if .IsDir}}
-						<td data-order="-1">&mdash;</td>
-						{{- else}}
-						<td data-order="{{.Size}}">{{.HumanSize}}</td>
-						{{- end}}
-						<td class="hideable"><time datetime="{{.HumanModTime "2006-01-02T15:04:05Z"}}">{{.HumanModTime "01/02/2006 03:04:05 PM -07:00"}}</time></td>
-						<td class="hideable"></td>
-					</tr>
-					{{- end}}
-					</tbody>
-				</table>
-			</div>
-		</main>
-		<footer>
-			Served with <a rel="noopener noreferrer" href="https://caddyserver.com">Caddy</a>
-		</footer>
-		<script>
-			var filterEl = document.getElementById('filter');
-			filterEl.focus({ preventScroll: true });
-
-			function initFilter() {
-				if (!filterEl.value) {
-					var filterParam = new URL(window.location.href).searchParams.get('filter');
-					if (filterParam) {
-						filterEl.value = filterParam;
-					}
-				}
-				filter();
-			}
-
-			function filter() {
-				var q = filterEl.value.trim().toLowerCase();
-				var elems = document.querySelectorAll('tr.file');
-				elems.forEach(function(el) {
-					if (!q) {
-						el.style.display = '';
-						return;
-					}
-					var nameEl = el.querySelector('.name');
-					var nameVal = nameEl.textContent.trim().toLowerCase();
-					if (nameVal.indexOf(q) !== -1) {
-						el.style.display = '';
-					} else {
-						el.style.display = 'none';
-					}
-				});
-			}
-
-			function localizeDatetime(e, index, ar) {
-				if (e.textContent === undefined) {
-					return;
-				}
-				var d = new Date(e.getAttribute('datetime'));
-				if (isNaN(d)) {
-					d = new Date(e.textContent);
-					if (isNaN(d)) {
-						return;
-					}
-				}
-				e.textContent = d.toLocaleString([], {day: "2-digit", month: "2-digit", year: "numeric", hour: "2-digit", minute: "2-digit", second: "2-digit"});
-			}
-			var timeList = Array.prototype.slice.call(document.getElementsByTagName("time"));
-			timeList.forEach(localizeDatetime);
-		</script>
-	</body>
-</html>`
+//go:embed browsetpl.html
+var defaultBrowseTemplate string
```

Comment:
Could we just move this line to `browse.go`? This file seems redundant now.

Reply:
Good idea, I updated the PR.